src/utils/utils.js

```diff
@@ -0,0 +1,27 @@
+export const getScrollbarWidth = () => {
+  const inner = document.createElement('p');
```

Comment:
Looking weird.

Reply:
Not for me: simplify that modal code and styling to pass requirements:
1. full width with background
2. UNABLE to scroll Y when modal is opened